src/librustc_trans/back/write.rs

```diff
@@ -664,8 +665,12 @@ pub fn run_passes(sess: &Session,
         if sess.opts.cg.codegen_units == 1 {
             // 1) Only one codegen unit.  In this case it's no difficulty
             //    to copy `foo.0.x` to `foo.x`.
-            fs::copy(&crate_output.with_extension(ext),
-                     &crate_output.path(output_type)).unwrap();
+            let input_path = &crate_output.with_extension(ext);
+            let output_path = &crate_output.path(output_type);
+            if let Err(e) = fs::copy(input_path, output_path) {
+                sess.err(&format!("could not copy {:?} to {:?}: {}",
+                                  input_path, output_path, e.description()));
```

Comment:
I think this may wish to rather just print `e` itself rather than just the `description` (as it has more information)